roles/ceph-mon/defaults/main.yml

```diff
@@ -10,6 +10,7 @@ fetch_directory: fetch/
 mon_group_name: mons
 
 # ACTIVATE BOTH FSID AND MONITOR_SECRET VARIABLES FOR NON-VAGRANT DEPLOYMENT
+# Comment out fsid when using mon_containerized_deployment_with_kv

```

Comment:
That is why i add {% if not mon_containerized_deployment_with_kv%}fsid = {{ fsid }}{% endif %} in ceph.conf so etcd populate will be the only one which  generate fsid.